pkg/bwagreement/server_test.go

```diff
@@ -152,3 +156,22 @@ func (TS *TestServer) Stop() {
 	}
 	TS.grpcs.Stop()
 }
+
+// call this function to copy signature and data into postgres db
+func readSampleDataFromPsdb() (map[string][]*psdb.Agreement, error) {
```

Comment:
Let's fill the database with actual data.

Reply:
Am bit not clear on the suggestion.

Below is the thought process of testing server.go's BandwidthAgreements()

Helper function readSampleDataFromPsdb() --> Implements reading from Piecestore's 'bandwidth_agreements' table and returns all the entries from that table as a slice. In TestBandwidthAgreements () test case I marshal the needed data(renter's signature and data) and send it through GRPC to satellite's bw agreement Postgres database ('pointerdb') and makes an entry into a table 'bwagreements'. This is test case tests server.go's BandwidthAgreements() which tests the creation of entry into Postgres table.

Reply:
yep I see your point...making mod